PyInstaller/depend/imphook.py

```diff
@@ -285,7 +285,9 @@ def __init__(self, module_graph, module_name, hook_filename,
 
         # Safety check, see above
         global HOOKS_MODULE_NAMES
-        assert self.hook_module_name not in HOOKS_MODULE_NAMES
+        assert self.hook_module_name not in HOOKS_MODULE_NAMES, \
+            'Hook Conflict. Please remove the following custom hook: ' \
+            '{hook}'.format(hook=self._hook_name)
```

Comment:
A quick question -- it `self._hook_name` correct? I can't find this being set anywhere in the code. Have you tested that this produces the correct error message?

Reply:
@bjones1 Umm? I meant to type `self.hook_module_name`. Sometimes autocomplete isn't helpful.